tests/flows/erc20_to_native_token_bridge.go

```diff
@@ -59,45 +61,69 @@ func ERC20ToNativeTokenBridge(network interfaces.LocalNetwork) {
 	Expect(err).Should(BeNil())
 	log.Info("Example ERC20 Contract Address: " + exampleERC20ContractAddress.Hex())
 
-	// Deploy the contracts
-	// Both contracts in this test will be deployed to 0x3405506b3711859c5070949ed9b700c7ba7bf750,
-	// though they do not necessarily have to be deployed at the same address, each contract needs
-	// to know the address of the other.
-	// The nativeTokenDestination contract must be added to "adminAddresses" of "contractNativeMinterConfig"
-	// in the genesis file for the subnet. This will allow it to call the native minter precompile.
-	erc20TokenSourceAbi, err := erc20tokensource.ERC20TokenSourceMetaData.GetAbi()
-	Expect(err).Should(BeNil())
-	utils.DeployContract(
-		ctx,
-		ERC20TokenSourceByteCodeFile,
-		deployerPK,
-		sourceSubnet,
-		erc20TokenSourceAbi,
-		sourceSubnet.TeleporterRegistryAddress,
-		destSubnet.BlockchainID,
-		bridgeContractAddress,
-		exampleERC20ContractAddress,
-	)
+	{
+		// deployerAddress needs 10 eth (1e19 wei) to deploy the contract, plus some for gas for other transactions
```

Comment:
```suggestion
		// Fund the deployer address with sufficient native tokens (10 eth = 1e20 wei) on the source chain to deploy the contract and send a number of transfer transactions.
```

I think the `10 eth` you're referring to here is the amount needed to deploy the `TeleporterMessenger` contract using Nick's method. This should be much lower because it uses an accurate gas estimation and dynamic fee txs.